Makefile

```diff
@@ -0,0 +1,35 @@
+MODULE := github.com/mongodb-forks/drone-helm3
+CMD_NAME ?= drone-helm
+VERSION := $(shell grep '^const Version =' internal/version/version.go | cut -d\" -f2)
+
+RUN ?= .*
+PKG ?= ./...
+.PHONY: test
+test: ## Run tests in local environment
+	golangci-lint run --timeout=5m $(PKG)
+	go test -cover -run=$(RUN) $(PKG)
+
+.PHONY: docker
+docker: ## Build local development docker image with cached go modules, builds, and tests
+	@docker build -f build/Dockerfile-test -t $(CMD_NAME):latest .
+
+.PHONY: docker-test
+docker-test: ## Run tests using local development docker image
+	@docker run -v $(shell pwd):/go/src/$(MODULE):delegated $(CMD_NAME) make test RUN=$(RUN) PKG=$(PKG)
```

Comment:
This command doesn't work when run with the provided `Dockerfile`.

Reply:
It ran using the Dockerfile-test file